setup.py

```diff
@@ -31,9 +31,10 @@
             'restructuredtext-lint',
             'Pygments',
             'coverage-conditional-plugin',
+            'tox~=3.28'
         ],
-        'mysql': ['mysql-connector-python-rf==2.2.2'],
-        'postgresql': ['psycopg2==2.8.4'],
+        'mysql': ['mysql-connector-python-rf>=2.2.2'],
+        'postgresql': ['psycopg2>=2.8.4'],
```

Comment:
I don't exactly recall the Python version that was building wheels. I didn't think the versions were all that important for validating proper execution. I can look into this deeper if necessary.